OpenSim/Simulation/SimbodyEngine/SimbodyEngine.cpp

```diff
@@ -978,6 +978,16 @@ void SimbodyEngine::convertRadiansToDegrees(Storage &rStorage) const
     scaleRotationalDofColumns(rStorage, SimTK_RADIAN_TO_DEGREE);
     rStorage.setInDegrees(true);
 }
+
+void SimbodyEngine::convertRadiansToDegrees(TimeSeriesTable& table) const {
+    constexpr double PI{3.141592653589793};
+    constexpr double RAD_TO_DEG{180 / PI};
+
+    table.updMatrix() *= RAD_TO_DEG;
+    table.removeTableMetaDataKey("inDegrees");
+    table.addTableMetaData("inDegrees", std::string{"yes"});
```

Comment:
I fixed it.
* Added an overload of `SimbodyEngine::scaleRotationalDofColumns` for `TimeSeriesTable` that does the same thing as the other overload.
* Now using the correct scale factor.

I think I got away with this previously because all the columns of the table were rotational dof and so comparison tests passed.